fdbserver/LogSystem.h

```diff
@@ -765,12 +765,23 @@ struct LogPushData : NonCopyable {
 	// copy written_tags, after filtering, into given set
 	void saveTags(std::set<Tag>& filteredTags) const {
 		for (const auto& tag : written_tags) {
-			if (!tag.isNonPrimaryTLogType()) {
```

Comment:
Why is this condition not needed any more?

Reply:
this is a function I added to get the set of tags a batch of transactions is directed to. Prior, we skipped cache tags (we only included locations >0). I think we need to include cache tags (location -1).